lib/events/event.ex

```diff
@@ -0,0 +1,74 @@
+defmodule ElixirTools.Events.Event do
```

Comment:
`Event` or `EventHandler` as in the README you specify `This module is responsible of handling events ...`

With current name looks like this module is an event structure, rather than a handler to deal with events data already created.

Reply:
it's the event structure, the handling is done in the entire section. I updated the readme to reflect this better.